modules/security-context-constraints-psa-about.adoc

```diff
@@ -0,0 +1,71 @@
+// Module included in the following assemblies:
+//
+// * authentication/understanding-and-managing-pod-security-admission.adoc
+// * operators/operator_sdk/osdk-complying-with-psa.adoc
+
+:_content-type: CONCEPT
+[id="security-context-constraints-psa-about_{context}"]
+= About pod security admission
+
+{product-title} includes link:https://kubernetes.io/docs/concepts/security/pod-security-admission[Kubernetes pod security admission]. Pods that do not comply with the pod security admission defined globally or at the namespace level are not admitted to the cluster and cannot run.
+
+Globally, the `privileged` profile is enforced, and the `restricted` profile is used for warnings and audits.
+
+To configure the pod security admission settings at the namespace level, set a label on the namespace in the following format: `pod-security.kubernetes.io/<mode>: <profile>`, where `<mode>` is the mode to configure and `<profile>` is the profile to use for that mode.
+
+[id="psa-modes_{context}"]
+== Pod security admission modes
+
+You can configure the following pod security admission modes for a namespace:
+
+.Pod security admission modes
+[cols="1,2,3a",options="header"]
+|===
+|Mode
+|Label
+|Description
+
+|`enforce`
+|`pod-security.kubernetes.io/enforce`
+|Rejects a pod from admission if it does not comply with the set profile.
+
+|`audit`
+|`pod-security.kubernetes.io/audit`
+|Logs audit events if a pod does not comply with the set profile.
+
+|`warn`
+|`pod-security.kubernetes.io/warn`
+|Displays warnings if a pod does not comply with the set profile.
+|===
+
+[id="psa-profiles_{context}"]
+== Pod security admission profiles
+
+You can set each of the pod security admission modes to one of the following profiles:
+
+.Pod security admission profiles
+[cols="1,3a",options="header"]
+|===
+|Profile
+|Description
+
+|`privileged`
+|Least restrictive policy; allows for known privilege escalation.
+
+|`baseline`
+|Minimally restrictive policy; prevents known privilege escalations.
+
+|`restricted`
+|Most restrictive policy; follows current pod hardening best practices.
+|===
+
+[id="psa-exemptions_{context}"]
+== Pod security admission exemptions
+
+The following system namespaces are exempt from pod security admission control:
+
+* `default`
+* `kube-public`
+* `kube-system`
+
+Pods in these namespaces are created even if they would have otherwise been rejected due to the set policy.
```

Comment:
That's not accurate, the namespaces are not exempt, we're only forcing the profile to be `privileged` there.

Reply:
Okay thanks. I think I had formed that from the PSA exemptions section here: https://kubernetes.io/docs/concepts/security/pod-security-admission/#exemptions

But if they aren't really exempt, we're just forcing privileged always, then I'll look into rewording this section.